platform/ui/src/assets/styles/styles.css

```diff
@@ -1,6 +1,11 @@
 /* CUSTOM OHIF SCROLLBAR */
 .ohif-scrollbar {
   scrollbar-color: #3E5967 transparent;
+  overflow-y: hidden;
```

Comment:
As requested, make this hidden unless hovering.
@sedghi - I did this for both the measurements and the thumbnail area, do you want me to do that just for thumbnails?